src/msxswagger/templates/swagger-sso-redirect.html

```diff
@@ -0,0 +1,125 @@
+<!doctype html>
+<html lang="en-US">
+<head><title>SSO Redirect</title></head>
+<body onload="run()">
+</body>
+</html>
+<script>
+    'use strict';
+    var InProgressRedirectKey = "CurrentRedirectSSO";
+    var SsoStateKeyPrefix = "sso-";
+
+    function run () {
+        var qp, arr;
+
+        if (/code|token|error/.test(window.location.hash)) {
+            qp = window.location.hash.substring(1);
+        } else {
+            qp = location.search.substring(1);
+        }
+
+        arr = qp.split("&")
+        arr.forEach(function (v,i,_arr) { _arr[i] = '"' + v.replace('=', '":"') + '"';})
+        qp = qp ? JSON.parse('{' + arr.join() + '}',
+            function (key, value) {
+                return key === "" ? value : decodeURIComponent(value)
+            }
+        ) : {}
+
+        var redirectCtx = loadRedirectContext({});
```

Comment:
If we inherited this code and it works as expected, we can keep it this way; however using `const` or `let` instead of `var` is the canonical way of declaring variables in JavaScript.